api/nakadi-event-bus-api.yaml

```diff
@@ -743,6 +743,56 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  /subscriptions:
+    post:
+      tags:
+        - managed-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        This endpoint creates a Subscription for EventType(s). The subscription is needed to be able to consume
+        events from EventType(s) in a high level way when Nakadi stores the offsets and manages the rebalancing of
+        consuming clients.
+        The subscription is identified by its parameters. If this endpoint is invoked several times with the same
+        subscription properties in body - the subscription will be created only once and for all calls it will
+        return the same subscription id.
+      parameters:
+        - name: subscription
+          in: body
+          description: Subscription to create
+          schema:
+            $ref: '#/definitions/Subscription'
+          required: true
+      responses:
+        '200':
```

Comment:
> Returning two different status could be against the mathematical definition of the term.

Idempotency in REST refers to the state change, not the response. Meaning, two idempotent requests should produce the same state on the server. It's totally valid to return e.g. 409 Conflict if a subscription already exists. The actual fact *the subscription is now present* is true, in both cases.